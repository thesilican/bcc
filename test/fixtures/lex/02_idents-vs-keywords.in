break char default double enum
hi hello_world HiHi __