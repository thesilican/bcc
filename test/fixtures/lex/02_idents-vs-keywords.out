TOKEN_KW "break"
TOKEN_KW "char"
TOKEN_KW "default"
TOKEN_KW "double"
TOKEN_KW "enum"
TOKEN_IDENT "hi"
TOKEN_IDENT "hello_world"
TOKEN_IDENT "HiHi"
TOKEN_IDENT "__"
